### PR TITLE
feat(oui-collapsible): collapsible component addition

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
UU-836

**Linked to**
https://github.com/ovh-ux/ovh-ui-angular/pull/173
https://github.com/ovh-ux/ovh-ui-kit-documentation/pull/94

### CODE REVIEW

**Comment by neolitec** [comment]
In collapsed state, the bottom border of the body is visible.

**Comment by ganeshkumar1989** [comment]
@neolitec , the body border in collapsed state has been removed

File: packages/oui-collapsible/_mixins.less [hunk 1]
```diff
@@ -0,0 +1,96 @@
+@import './_variables';
+
+#oui {
+  .collapsible(
```
Comment (AxelPeter) [inline packages/oui-collapsible/_mixins.less]: (optional) You can put your style directly in the main file. We stopped structuring our styles with this kind of mixin (called once) :)

Here for example: https://github.com/ovh-ux/ovh-ui-kit/blob/master/packages/oui-stepper/stepper.less

Comment (ganeshkumar1989) [reply packages/oui-collapsible/_mixins.less]: Sure. Thanks for the comment. I will follow this from my next PR onwards :)

**Pull request merged**